<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogErase</class>
 <widget class="QDialog" name="DialogErase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>222</width>
    <height>182</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Erase chip</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_Chip">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Chip type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="chip">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_VpinA9">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Erase voltage VPE:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pinE">
       <property name="text">
        <string>(pins A9, VPP)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_d">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>datasheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="d_VPE">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_s">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="s_VPE">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="SoftErase">
     <property name="text">
      <string>Software chip erase</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="EandR">
     <property name="text">
      <string>Run check for emptiness after erasing</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogErase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogErase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
